demo/plugins.txt

```diff
@@ -1,2 +1,6 @@
+# Development mode
 simple-theme-plugin:0.6
 login-theme:1.1
+# Common mode
+dark-theme:0.0.3
```

Comment:
I'm planning to add something to make development easier,
(needed for the blue ocean work :p )

Reply:
Yes. I am basically revering the login theme magic I was using for development in this demo